src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  transition-duration: all 1s;
+
+  &__image {
+    box-sizing: border-box;
+    height: 134px;
+    width: 160px;
+    margin-top: 32px;
+    margin-inline: 3px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    box-sizing: border-box;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    &-number {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-text-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-color;
+    }
+
+    &-cost {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__link {
+    box-sizing: border-box;
+    background-color: $blue-color;
+    height: 40px;
+    color: $white-color;
+    border-radius: 5px;
+    border: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    margin-top: 16px;
+    margin-bottom: 16px;
+    text-decoration: none;
+    text-align: center;
+    cursor: pointer;
+  }
+}
+
+.card:hover {
+  transform: scale(120%);
```

Comment:
The transform scale property can lead to layout shifts and is not recommended for performance reasons. Consider another way to highlight the card on hover, like changing the box shadow or border color.